adviser/base/argo-workflows/advise-demo.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: advise-demo
+spec:
+  templates:
+    - name: advise-demo
+      resubmitPendingPods: true
+      inputs:
+        parameters:
+          - name: "project-url"
+          - name: "fork-namespace"
+      container:
+        image: workflow-helpers
+        volumeMounts:
+          - name: ssh-config
+            mountPath: /home/user/.ssh
+            readOnly: true
+        env:
+          - name: THOTH_DEMO_PROJECT_URL
+            value: "{{inputs.parameters.project-url}}"
+          - name: THOTH_WORKFLOW_TASK
+            value: "thoth_advise_demo"
+          - name: THOTH_DEMO_FORK_NAMESPACE
+            value: "{{inputs.parameters.fork-namespace}}"  # CHNG: get value from config map or pass directly from Thoth common?
```

Comment:
@harshad16 I want to create a configmap entry for this value rather than making it a template parameter. Which configmap should I add it to?

Reply:
If it is dependent on cluster, it can go in the **configmap thoth** under **common** in each **overlays** https://github.com/thoth-station/thoth-application/tree/master/core/overlays
or else you can add it to the **base configmap thoth**